src/app/modules/submission/submission.utils.ts

```diff
@@ -51,18 +51,9 @@ const encryptFormFieldModeFilter = <T extends FormField>(
   responses: T[] = [],
 ) => {
   // To filter for autoreply-able fields.
-  return responses.filter((response) => {
-    if ([BasicField.Mobile, BasicField.Email].includes(response.fieldType))
-      return false
-    switch (response.fieldType) {
-      case BasicField.Mobile:
-        return response.isVerifiable
-      case BasicField.Email:
-        return response.autoReplyOptions.hasAutoReply || response.isVerifiable
-      default:
-        return false
-    }
-  })
```

Comment:
this is an intended change in our app, we should update tests for this behaviour instead

Reply:
ok got it! should we remove this part though?

>` if ([BasicField.Mobile, BasicField.Email].includes(response.fieldType))
       return false
`

since including it means we won't get to the switch case statements for `BasicField.Mobile` and `BasicField.Email`, i.e.this function always returns an empty array

Reply:
Wait a second was that a bug!!! 😭

Reply:
yaaa 🐛 🐞 😢

Reply:
can double check if behaviour changes from expected listed in https://github.com/opengovsg/FormSG/pull/4001 if we remove the false check?

Reply:
commented out the false check, checked if I was able to

- view an individual storage mode form response
- navigate to the next/previous response
- download individual attachment

all of the above work!

I'm not able to receive an email auto-reply though, investigating further

Reply:
try submitting verified and unverified mobile/email fields just to be safe

Reply:
btw is autoreplyfor mobile implemented? i can't seem to find it heh